pulp_rpm/tests/functional/api/test_crud_content_unit.py

```diff
@@ -45,173 +34,86 @@ class ContentUnitTestCase(unittest.TestCase):
         """Attempt to update a content unit using HTTP PUT.
 
         This HTTP method is not supported and a HTTP exception is expected.
         """
-        attrs = copy.deepcopy(RPM_PACKAGE_DATA)
-        attrs.update({'name': utils.uuid4()})
-        with self.assertRaises(HTTPError) as exc:
-            self.client.put(self.content_unit['pulp_href'], attrs)
-        self.assertEqual(exc.exception.response.status_code, 405)
+        attrs = gen_rpm_content_attrs(self.artifact, RPM_PACKAGE_FILENAME2)
+        with self.assertRaises(AttributeError) as exc:
+            self.rpm_content_api.update(self.content_unit["pulp_href"], attrs)
+        msg = "object has no attribute 'update'"
+        self.assertIn(msg, exc.exception.args[0])
 
-    @skip_if(bool, 'content_unit', False)
+    @skip_if(bool, "content_unit", False)
     def test_04_delete(self):
         """Attempt to delete a content unit using HTTP DELETE.
 
         This HTTP method is not supported and a HTTP exception is expected.
         """
-        with self.assertRaises(HTTPError) as exc:
-            self.client.delete(self.content_unit['pulp_href'])
-        self.assertEqual(exc.exception.response.status_code, 405)
-
-
-class DuplicateContentUnit(unittest.TestCase):
-    """Attempt to create a duplicate content unit.
-
-    This test targets the following issues:
-
-    *  `Pulp #4125 <https://pulp.plan.io/issue/4125>`_
-    """
-
-    @classmethod
-    def setUpClass(cls):
-        """Create class-wide variables."""
-        cls.cfg = config.get_config()
-        cls.client = api.Client(cls.cfg, api.json_handler)
-
-    @classmethod
-    def tearDownClass(cls):
-        """Clean created resources."""
-        delete_orphans(cls.cfg)
-
-    def test_raise_error(self):
-        """Create a duplicate content unit using same artifact and filename."""
-        delete_orphans(self.cfg)
-        files = {'file': utils.http_get(RPM_UNSIGNED_URL)}
-        artifact = self.client.post(ARTIFACTS_PATH, files=files)
-        attrs = {
-            'artifact': artifact['pulp_href'],
-            'relative_path': RPM_PACKAGE_FILENAME
-        }
-
-        # create first content unit.
-        self.client.using_handler(api.task_handler).post(RPM_CONTENT_PATH, attrs)
-
-        with self.assertRaises(TaskReportError) as ctx:
-            # using the same attrs used to create the first content unit.
-            api.Client(self.cfg, api.task_handler).post(
-                RPM_CONTENT_PATH,
-                attrs
-            )
-
-        keywords = (
-            'name',
-            'epoch',
-            'version',
-            'release',
-            'arch',
-            'checksum_type',
-            'pkgId',
-        )
-
-        for key in keywords:
-            self.assertIn(
-                key.lower(),
-                ctx.exception.task['error']['description'].lower(),
-                ctx.exception.task['error']
-            )
-
-    def test_second_unit_raises_error(self):
```

Comment:
I think this test wasn't converted

Reply:
it is 'test_05_duplicate_raise_error' in 'ContentUnitTestCase' class

Reply:
this test uses `RPM_SIGNED_URL2` and it is not used on `test_05_duplicate_raise_error`,
I believe `test_05_duplicate_raise_error` is the converted version of the previous test: `test_raise_error`

Reply:
true, added